flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendCancelTest.java

```diff
@@ -39,13 +47,27 @@
 /**
  * Tests for the CANCEL command.
  */
+@RunWith(Parameterized.class)
 public class CliFrontendCancelTest extends TestLogger {
 
+	@Parameterized.Parameters(name = "Mode = {0}")
```

Comment:
Everything about introducing this parameter is duplicated in all the test classes. I would create a common super class that does this parameter iteration and implements the `getCli()` and `getConfiguration()` methods, instead of having them as static methods.